metadata/pkg/validator/AWSQueryValidator.go

```diff
@@ -0,0 +1,31 @@
+package validator
```

Comment:
just to validate region, why do we need seprate file.

Reply:
moved all validation logic to validator file